src/.goreleaser.yml

```diff
@@ -1,27 +1,71 @@
-# This is an example .goreleaser.yml file with some sensible defaults.
-# Make sure to check the documentation at https://goreleaser.com
+project_name: knoxAutoPolicy
+
 before:
   hooks:
     # You may remove this if you don't use go modules.
     - go mod tidy
     # you may remove this if you don't need go generate
     - go generate ./...
+
 builds:
-  - env:
-      - CGO_ENABLED=1
+  - binary: "opt/knoxAutoPolicy/src/knoxAutoPolicy"
+    id: knoxAutoPolicy
     goos:
       - linux
         #- windows
         #- darwin
     goarch:
       - amd64
```

Comment:
Kindly include **arm64** as well

Reply:
I think we can add it once we support KubeArmor too. :grin: